pkg/features/features.go

```diff
@@ -101,4 +101,9 @@ const (
 	// owner: @petersutter @xrstf
 	// alpha: v1.28.0
 	RotateSSHKeypairOnMaintenance featuregate.Feature = "RotateSSHKeypairOnMaintenance"
+
+	// DenyInvalidExtensionResources causes the seed-admission-controller to allow invalid extension resources.
```

Comment:
Ups, you might want to correct this sentence 😄 
```suggestion
	// DenyInvalidExtensionResources causes the seed-admission-controller to deny invalid extension resources (instead of just logging validation errors).
```